### PR TITLE
Adding README.md

### LINKED ISSUE #9: Missing README
This repository is missing a README. Please add a `README.md` file covering the following topics:

- [x] Purpose of this extension
- [x] How to install
- [x] Compatibility requirements (TYPO3/PHP)

### PR DESCRIPTION
**Pull request by dermodmaster**
Resolves #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Mittwald Cache (APCu & OpCodeCache) widget for the TYPO3 dashboard  
+
+## Purpose of this extension
+This extension allows to draw a doughnut chart widget in the TYPO3 dashboard, about the memory usage of your cache system. 
+It will show how much memory is in use and how much is free.
```
Comment (martin-helmich) [inline README.md:5]: Maybe emphasize again that this is about _APCu and OPCache_ memory usage, and NOT memory usage in general?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (martin-helmich) [inline README.md:6]: This would be the ideal place in include a screenshot 😉

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	 2. After that, above you will see a dropdown with the default selection **Installed Extensions**. 
+	 3. Here you need to select **Get Extensions**. 
+	 4. Please search for "**mw_cache_widget**" (If nothing appears, maybe you need to update your extension list with the button **Update now** on the right and try this step again).
+	 5.  Finally press the button with the cloud symbol (**Import and Install**).
+ 
+ 2. **Manual installation**
+	 1. [Download this extension as a zip file from here](https://extensions.typo3.org/extension/mw_cache_widget/).
+	 2.  Please login into your TYPO3 Backend and select **Extensions** in the side menu. 
+	 3. Click on the upload button under the dropdown menu above.
+	 4. Select the zip file and click on **Upload!**.
```
Comment (martin-helmich) [inline README.md:20]: Just a side note: Please remember to also document the installation via Composer when #1 is resolved.

**Review by martin-helmich — CHANGES_REQUESTED** [review]

**Comment by derhansen** [comment]
Resolved with ecd28b8899d294af0c19cb704a8058a68c8908e3

**Pull request closed without merging**